libsplinter/src/node_id/store/file/mod.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! NodeIdStore that supports the 0.4 node_id file format.
+
+use std::fs;
+use std::path::PathBuf;
+
+use super::NodeIdStore;
+use super::NodeIdStoreError;
+
+/// The 0.4 node_id file is depricated, this exists to help migrate the node_id.
```

Comment:
depricate -> deprecated

But more specifically, change it to "soft-deprecated", as Rust used to describe `Error::description()` until it was deprecated via the metadata - since we're not adding that metadata here yet.

I would also change this sentence to be part of the doc detail and the summary line should just be "A NodeIdStore backed by a file".